src/sys/select.rs

```diff
@@ -68,11 +87,18 @@ mod ffi {
     }
 }
 
-pub fn select(nfds: c_int,
-              readfds: Option<&mut FdSet>,
+pub fn select(readfds: Option<&mut FdSet>,
               writefds: Option<&mut FdSet>,
               errorfds: Option<&mut FdSet>,
               timeout: Option<&mut TimeVal>) -> Result<c_int> {
+    // calculate nfds parameter, the largest fd we want to select on
+    let nfds = readfds.iter()
+        .chain(writefds.iter())
```

Comment:
Will do, but please consider moving to the proposed RFC style. Most editors make it surprisingly hard to indent code like this.